### PR TITLE
Google.AndroidM3Emulator version m3-rc20a

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106155)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230507195651

**Comment by stephengillie** [comment]
This package runs from its local directory, but the symlink gives the error:
> Cannot locate system directory, pluse use -system <directory> to specify one

I think it needs to be run as `emulator -system %appdatalocal%\Microsoft\WinGet\Packages\Google.AndroidM3Emulator__DefaultSource\android_sdk_windows_m3-rc20a\tools\` but I'm unsure how to specify this argument for the symlink.

File: manifests/g/Google/AndroidM3Emulator/m3-rc20a/Google.AndroidM3Emulator.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Google.AndroidM3Emulator
+PackageVersion: m3-rc20a
+Installers:
+- InstallerUrl: https://dl.google.com/android/archives/android_sdk_windows_m3-rc20a.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
```
Comment (stephengillie) [inline manifests/g/Google/AndroidM3Emulator/m3-rc20a/Google.AndroidM3Emulator.installer.yaml:10]:
```suggestion
  NestedInstallerType: portable
```

Comment (stephengillie) [reply manifests/g/Google/AndroidM3Emulator/m3-rc20a/Google.AndroidM3Emulator.installer.yaml:10]: When set to portable, this package runs from its local directory, but the symlink gives the error:
> Cannot locate system directory, pluse use -system <directory> to specify one

I think it needs to be run as `emulator -system %appdatalocal%\Microsoft\WinGet\Packages\Google.AndroidM3Emulator__DefaultSource\android_sdk_windows_m3-rc20a\tools\` but I'm unsure how to specify this argument for the symlink.

**Comment by stephengillie** [comment]
https://github.com/microsoft/winget-pkgs/pull/106155#discussion_r1187722917

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by Trenly** [comment]
@upintheairsheep - Support for zipped binaries has now been added and is available in the pipelines. This will require you to update your submission to ManifestVersion: 1.9.0 and set ArchiveBinariesDependOnPath: true in the installer node. To keep things clean, this PR is being closed. Please submit a new PR for the latest version of this package. Thank you for your contributions!

**Pull request closed without merging**